shoop_workbench/settings/base_settings.py

```diff
@@ -157,7 +158,7 @@
 
 SESSION_SERIALIZER = "django.contrib.sessions.serializers.PickleSerializer"
 
-SHOOP_PRICING_MODULE = "default_pricing"
+SHOOP_PRICING_MODULE = "discount_pricing"
```

Comment:
Product requirements state that it is needed to have discounts, so I think it is necessary.